website/content/en/preview/upgrade-guide.md

```diff
@@ -102,6 +102,10 @@ Snapshot releases are tagged with the git commit hash prefixed by the Karpenter
 
 ## Released Upgrade Notes
 
+### Upgrading to v0.30.0+
+
+* Karpenter will now statically drift on both Provisioner and AWSNodeTemplate Fields. For Provisioner Static Drift, the annotation `karpenter.sh/provisioner-hash` must be present on both the Provisioner and Machine. For AWSNodeTemplate drift, the annotation `karpenter.k8s.aws/nodetemplate-hash` annotation must be present on the AWSNodeTemplate and Machine. Karpenter will not add these annotations to pre-existing nodes so these nodes will need to be recycled one time for the annotations to be added.
```

Comment:
Should you link to the section that define static drift here? Without knowing what that actually means, it's unclear to a user what we are drifting on.